sdk/monitor/Azure.Monitor.OpenTelemetry.TelemetryClient/tests/Azure.Monitor.OpenTelemetry.TelemetryClient.Tests/Azure.Monitor.OpenTelemetry.TelemetryClient.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="xunit" />
+    <PackageReference Include="WireMock.Net" />
```

Comment:
What is WireMock being used for here?   From its repository README, it provides the same general record/playback proxying that the Azure Core test framework does.    Unless there's some other feature needed, we should be using the test framework for this functionality.